openquakeplatform_taxtweb/utils/taxonomy2human.py

```diff
@@ -0,0 +1,338 @@
+        for item in arr:
+            d[item["id"]] = item
+    return d
+
+
+def llrs2human(blk, no_unknown=False):
+    blk_out = ""
+    blk_err = ""
+    atoms = blk.split('+')
+
+    llrs_types = arrdicts_flatten(llrs_type_grp)
+    llrs_ducts = arrdicts_flatten(llrs_duct_grp)
+
+    for atom in atoms:
+        if atom in Taxonomy.UNKNOWN_ATOMS and no_unknown:
+            continue
+
+        if atom in llrs_types:
+            if blk_out:
+                blk_out += '; '
+            blk_out += ('Type lateral load-resisting system: ' +
+                        llrs_types[atom]['desc'])
+        elif atom in llrs_ducts:
+            if blk_out:
+                blk_out += '; '
+            blk_out += 'System ductility: ' + llrs_ducts[atom]['desc']
+        else:
+            blk_err += ' ' + atom + ' not found'
+
+    return (blk_out, blk_err)
+
+
+def lmat2human(blk, no_unknown=False):
+    blk_out = ""
+    blk_err = ""
+    atoms = blk.split('+')
+
+    mat_erials = arrdicts_flatten([material])
+    mat_techs = arrdicts_flatten(mat_tech_grp)
+    mat_props = arrdicts_flatten(mat_prop_grp)
+    mat_teads = arrdicts_flatten(mat_tead_grp)
+
+    for atom in atoms:
+        if atom in Taxonomy.UNKNOWN_ATOMS and no_unknown:
+            continue
+
+        if atom in mat_erials:
+            if blk_out:
+                blk_out += '; '
+            blk_out += 'Material type: ' + mat_erials[atom]['desc']
+        elif atom in mat_techs:
+            if blk_out:
+                blk_out += '; '
+            blk_out += 'Material technology: ' + mat_techs[atom]['desc']
+        elif atom in mat_props:
+            if blk_out:
+                blk_out += '; '
+            blk_out += 'Material properties: ' + mat_props[atom]['desc']
+        elif atom in mat_teads:
+            if blk_out:
+                blk_out += '; '
+            blk_out += ('Material technology (additional): ' +
+                      mat_teads[atom]['desc'])
+        else:
+            blk_err += ' ' + atom + ' subblock not found'
+
+    return (blk_out, blk_err)
+
+
+def height2human(blk, no_unknown=False):
+    blk_out = ""
+    blk_err = ""
+    subblks = blk.split('+')
+
+    hei_aboveground = arrdicts_flatten([h_aboveground])
+    hei_belowground = arrdicts_flatten([h_belowground])
+    hei_abovegrade = arrdicts_flatten([h_abovegrade])
+    hei_slope = arrdicts_flatten([h_slope])
+
+    for subblk in subblks:
+        sub2blks = subblk.split(':')
+        atom = sub2blks[0]
+        if atom in Taxonomy.UNKNOWN_ATOMS and no_unknown:
+            continue
+
+        if atom in hei_aboveground:
+            pfx = 'Number of storeys above ground'
+            desc = hei_aboveground[atom]['desc']
+            sfx = ''
+        elif atom in hei_belowground:
+            pfx = 'Number of storeys below ground'
+            desc = hei_belowground[atom]['desc']
+            sfx = ''
+        elif atom in hei_abovegrade:
+            pfx = 'Number of storeys above grade'
```

Comment:
I think this should be 'Height of ground floor above grade in meters' not number of storeys